jme3-plugins-json-gson/src/main/java/com/jme3/plugins/gson/GsonElement.java

```diff
@@ -46,6 +48,22 @@ class GsonElement implements JsonElement {
         this.element = element;
     }
 
+    @Override
+    public int hashCode() {
+        int hash = 7;
```

Comment:
All of this weird hash magic seems unnecessary.  Just return element.hashCode().

Reply:
fixes applied (magic hash)...